configs/test/bot/setup/android.bash

```diff
@@ -24,8 +43,7 @@ if [ -z "$GOOGLE_APPLICATION_CREDENTIALS" ]; then
 fi
 
 if [ -z "$ANDROID_SERIAL" ]; then
-  echo "\$ANDROID_SERIAL is not set."
-  exit 1
+  echo "\$No ANDROID_SERIAL set. Will automatically detect devices and start ClusterFuzz for each."
```

Comment:
Why not move this block at line 114, does not need to be here.

Reply:
Hmm, good point. Moved the echos to the existing blocks there.